Classes/Domain/Model/PictureConfiguration.php

```diff
@@ -135,7 +135,7 @@ public function webpShouldBeAddedBeforeSrcset(): bool
 
     public function webpShouldBeAddedAfterSrcset(): bool
     {
-        return $this->addWebp && $this->addSources;
+        return $this->addWebp || $this->addSources;
```

Comment:
mh, this do not mirror the method name `webpShouldBeAddedAfterSrcset`, why should webp be added if addWebp is false?

Reply:
This change is not needed IMHO